src/types/api.ts

```diff
@@ -0,0 +1,648 @@
+interface BaseListResponse {
+  meta: object;
+  jsonapi: object;
+  links: object;
+  included?: Array<T>;
+}
```

Comment:
The `T` generic is missing here. This suggestion should fix it.
```suggestion
interface BaseListResponse<T> {
  meta: object;
  jsonapi: object;
  links: object;
  included?: Array<T>;
}
```

Reply:
I wondered about this. Would using `included?: Array<any>;` be better or worse?

Reply:
IMO worse. You can use `any` if you want but that means the types are more vague.
If you want you could do something like this instead if you want to have a default type for `T`?
```ts
interface BaseListResponse<T = unknown> {
  meta: object;
  jsonapi: object;
  links: object;
  included?: Array<T>;
}
```